packages/core/test/json.test.ts

```diff
@@ -88,7 +88,8 @@ describe('json', () => {
             }
           }
         }
-      }
+      },
+      output: {}
```

Comment:
This could look slightly less odd
```suggestion
      output: { result: 42 }
```